### PR TITLE
364 gv edge overlap overlapping degree

### PR DESCRIPTION
**Pull request by giovannivolpe**
@annacanal please, revise carefully the changes

### CODE REVIEW

File: braph2/graph/measures/OverlappingDegree.m [hunk 1]
```diff
@@ -80,19 +78,19 @@
             %
             % See also getClass, getDescription. 
             
-            name = 'Overlapping-Degree';
+            name = 'Overlapping degree';
```
[the remark this answers is not in the document]
Comment (egolol) [reply braph2/graph/measures/OverlappingDegree.m:81]: I think underscore _  is better, both - and ' ' (space) sometimes are problematic with software.

[the remark this answers is not in the document]
Comment (annacanal) [reply braph2/graph/measures/OverlappingDegree.m:81]: True. Then let's all adopt '_'

File: braph2/graph/measures/DegreeOverlap.m [hunk 2]
```diff
@@ -1,49 +1,46 @@
 classdef DegreeOverlap < EdgeOverlap
-    % DegreeOverlap Overlapping Degree measure
-    % DegreeOverlap provides the overlapping degree of a node for binary 
-    % undirected (BU) and weighted undirected (WU) graphs. It is calculated 
-    % as the number of edges connected to the node. 
+    % DegreeOverlap Edge overlap measure
```
Comment (annacanal) [inline braph2/graph/measures/DegreeOverlap.m:2]: DegreeOverlap **Degree** overlap measure

File: braph2/graph/measures/OverlappingDegree.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (annacanal) [reply braph2/graph/measures/OverlappingDegree.m:81]: Agreed with Giovanni to go with spaces. Use of class name (getClass()) when spaces are a problem.

File: braph2/graph/measures/EdgeOverlap.m [hunk 3]
```diff
@@ -77,19 +73,19 @@
             %
             % See also getClass, getDescription. 
             
-            name = 'Edge-Overlap';
+            name = 'Edge overlap';
```
[the remark this answers is not in the document]
Comment (annacanal) [reply braph2/graph/measures/EdgeOverlap.m:76]: Solved. We go with spaces

File: braph2/graph/measures/DegreeOverlap.m (hunk 2 above)
Comment (giovannivolpe) [reply braph2/graph/measures/DegreeOverlap.m]: done

File: braph2/graph/measures/DegreeOverlap.m [hunk 4]
```diff
@@ -0,0 +1,142 @@
+classdef DegreeOverlap < EdgeOverlap
+    % DegreeOverlap Edge overlap measure
+    % DegreeOverlap provides the degree overlap of a node for binary 
+    % undirected (BU) and weighted undirected (WU) multiplexes. 
+    % 
+    % It is calculated as the number of edges connected to the node in all
+    % layers.
+    % 
+    % DegreeOverlap methods:
+    %   DegreeOverlap           - constructor
```
[the remark this answers is not in the document]
Comment (giovannivolpe) [reply braph2/graph/measures/DegreeOverlap.m]: done

File: braph2/graph/measures/DegreeOverlap.m [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    %   getMeasure                  - returns the degree class
+    %   getCompatibleGraphList      - returns a list of compatible graphs
+    %   getCompatibleGraphNumber    - returns the number of compatible graphs
+    %
+    % See also Measure, EdgeOverlap, MultiplexGraphBU, MultiplexGraphWU.
+    
+    methods
+        function m = DegreeOverlap(g, varargin)
+            % DEGREEOVERLAP(G) creates degree overlap with default properties.
+            % G is a undirected multiplex (i.e., an instance of
```
[the remark this answers is not in the document]
Comment (giovannivolpe) [reply braph2/graph/measures/DegreeOverlap.m]: done

**Pull request merged**